.fixtures.yml

```diff
@@ -14,12 +14,12 @@ fixtures:
       ref: "1.0.0"
   repositories:
     "stdlib":
-      "repo": "git://github.com/puppetlabs/puppetlabs-stdlib.git"
+      "repo": "git@github.com:puppetlabs/puppetlabs-stdlib.git"
```

Comment:
Can we switch these to HTTPS as opposed to utilizing SSH?

```
"repo": "https://github.com:puppetlabs/puppetlabs-stdlib.git"
```

Reply:
I have switched to the https version as per your recommendation. see file difference.